### PR TITLE
add expo template for aws amplify storage upload

### PR DESCRIPTION
**Pull request by ykbryan**
Providing an expo template for AWS users and an example to do storage upload.
Try: https://exp.host/@ykbryan/aws-storage-upload
Working repo: https://github.com/ykbryan/expo-example-aws-storage-upload

### CODE REVIEW

File: with-aws-storage-upload/package.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
+{
+  "main": "index.js",
+  "scripts": {
+    "android": "react-native run-android",
+    "ios": "react-native run-ios",
+    "web": "expo start --web",
+    "start": "react-native start",
+    "test": "jest"
+  },
```
Comment (EvanBacon) [inline with-aws-storage-upload/package.json:9]:
```suggestion
```
No scripts

File: with-aws-storage-upload/package.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
+{
+  "main": "index.js",
```
Comment (EvanBacon) [inline with-aws-storage-upload/package.json:2]: This file doesn't exist

File: with-aws-storage-upload/package.json [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    "react-native-reanimated": "~1.7.0",
+    "react-native-screens": "~2.2.0",
+    "react-native-unimodules": "~0.9.0",
+    "react-native-web": "~0.11.7"
+  },
+  "devDependencies": {
+    "@babel/core": "~7.9.0",
+    "babel-jest": "~25.2.6",
+    "jest": "~25.2.6",
+    "react-test-renderer": "~16.9.0"
```
Comment (EvanBacon) [inline with-aws-storage-upload/package.json:32]: Not required for a minimal demo of amplify storage upload

File: with-aws-storage-upload/package.json [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+  },
+  "jest": {
+    "preset": "react-native"
+  },
+  "private": true
```
Comment (EvanBacon) [inline with-aws-storage-upload/package.json:37]:
```suggestion
```

File: with-aws-storage-upload/package.json [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+    "ios": "react-native run-ios",
+    "web": "expo start --web",
+    "start": "react-native start",
+    "test": "jest"
+  },
+  "dependencies": {
+    "@aws-amplify/auth": "^3.2.13",
+    "@aws-amplify/core": "^3.3.3",
+    "@aws-amplify/storage": "^3.2.6",
+    "expo": "~37.0.3",
```
Comment (EvanBacon) [inline with-aws-storage-upload/package.json:14]:
```suggestion
    "expo": "~38.0.0",
```

File: with-aws-storage-upload/package.json (hunk 2 above)
Comment (ykbryan) [reply with-aws-storage-upload/package.json]: Thanks for the review.  
I have updated the package.json

**Comment by ykbryan** [comment]
@EvanBacon Thanks for the review.
Tested privately in my repo as well - https://github.com/ykbryan/expo-example-aws-storage-upload

**Review by EvanBacon — APPROVED** [review]
LGTM, thanks

**Pull request merged**